### PR TITLE
Finish log-in log-out with devise gem

### PR DESCRIPTION
**Pull request by binhthanhnguyen**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 Rails.application.routes.draw do
+  devise_for :users
   root "static_pages#home"
-
   namespace :admin do
     root "subjects#index"
     resources :subjects
   end
```
Comment (ThuBM) [inline config/routes.rb]: add new line after this line

File: config/routes.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
+  devise_for :users
   root "static_pages#home"
-
   namespace :admin do
```
Comment (ThuBM) [inline config/routes.rb]: add new line above this line!

**Comment by thiennv57** [comment]
OK

**Comment by VuongLong** [comment]
ok

**Comment by ThuBM** [comment]
OK

**Pull request merged**